docs/verifiable-document/document-template.md

```diff
@@ -121,11 +123,13 @@ export const registry: TemplateRegistry<any> = {
 };
 ```
 
-### Multiple views for a Template
+### Multiple views for a template
+
+An OA document may have multiple views, each of them rendered in separate tabs. For example, an OA document that is a degree certificate may have the actual certificate as one view, and the transcript as another view in a single template. A demo with can be found at our [here](https://opencerts.io/?q={%22type%22:%22DOCUMENT%22,%22payload%22:{%22uri%22:%22https://opencerts.io/static/demo/mainnet.opencerts%22,%22permittedActions%22:[%22STORE%22],%22redirect%22:%22https://opencerts.io%22}}).
```

Comment:
`A demo with can be found at our`

either sth is missing, either sth must be removed :D

Reply:
thanks for pointing it out! it was a typo hahaha

Reply:
fixed